djgeojson/serializers.py

```diff
@@ -14,8 +14,15 @@
 
 from six import string_types, iteritems
 
+import django
+
 from django.db.models.base import Model
-from django.db.models.query import QuerySet, ValuesQuerySet
+
+try:
+    from django.db.models.query import QuerySet, ValuesQuerySet
+except ImportError:
+    from django.db.models.query import QuerySet
```

Comment:
I suggest that in that case you do that:
```
ValuesQuerySet = QuerySet
```

So that you don't have to test the django version in the middle of the class.